Library/ViewModels/CommentComposerViewModel.swift

```diff
@@ -28,11 +31,14 @@ public protocol CommentComposerViewModelOutputs {
   var avatarURL: Signal<URL?, Never> { get }
 
   /// Emits a string that should be put into the body text view.
-  var bodyText: Signal<String, Never> { get }
+  var bodyText: Signal<String?, Never> { get }
```

Comment:
Remind me why we emit the body text here? What is the scenario in which we will need to set the text in the textfield from the view model?

Reply:
So that we can clear the text view after posting cause at that point I'm also calling `resetInput` that resets the `bodyText` in the view model via `bodyTextDidChange(nil)`

Reply:
Oh right - cool, can we change it so that it's not updating the body text while we're typing? it seems unnecessary to type -> call input -> set text in a cycle. It would make more sense that when `resetInput` is called or the comment is posted, we emit a `nil` bodyText for the textview.

Reply:
Just seeing this. I'll look into that